pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -360,7 +364,27 @@ type S3StorageProvider struct {
 	// Acl represents access control permissions for this bucket
 	Acl string `json:"acl,omitempty"`
 	// SecretName is the name of secret which stores
-	// ceph object store access key and secret key.
+	// S3 compliant storage access key and secret key.
+	SecretName string `json:"secretName"`
+}
+
+// +k8s:openapi-gen=true
+// GcsStorageProvider represents the google cloud storage for storing backups.
+type GcsStorageProvider struct {
+	// ProjectNumber represents the project that organizes all your Google Cloud Platform resources
+	ProjectNumber string `json:"projectNumber"`
```

Comment:
I think it's better to use project id, the project number is seldom used. You can use `gcloud projects list` to get the available project lists. Ref: https://cloud.google.com/resource-manager/docs/creating-managing-projects

Reply:
The name here is to keeping consistent with the configuration of rclone. The actual value is the project id. To avoid ambiguity, I changed this to projectID.